XamlControlsGallery/Navigation/NavigationRootPage.xaml

```diff
@@ -84,7 +84,6 @@
                     PlaceholderText="Search"
```

Comment:
In addition to removing the RequestedTheme, I would also delete the Style setter. I think it's fine for this ASB to just have the default look and feel.

Reply:
Removed it in the latest commit.